external/fv3fit/fv3fit/keras/_models/packer.py

```diff
@@ -0,0 +1,74 @@
+from typing import Sequence, Mapping
```

Comment:
Can you move all the `ArrayPacker3D` logic here for now? That would ensure the diff doesn't touch `external/fv3fit/fv3fit/_shared/packer.py`